### ISSUE TITLE
Super Admin Permissions

### ISSUE THREAD

**Issue by velkymx** [ProcessMaker/bpm#1388]
After creating a new user, I immediately assign this user Admin permissions by selecting the "Make this user an Admin" checkbox. After clicking "Update," I return to the same user and view permissions. The "Make this user an Admin" checkbox is not selected. Furthermore, I suspect that this user is not truly an Admin even though this new user has access to the "Admin" section. I suggest this because this new user doesn't have access to the "Queue Management" page (the icon to access it does not display). But this user has some Admin access. None of the permission checkboxes are selected to know which permissions this new user actually has.

**Comment by milaendo** [comment]
I have followed these step to reproduce and cannot successfully reproduce this. I was able to make the new user, revisit the page and see their permisions. I was also able to log in as this new user and had full access to queue management and perform all other "super admin" tasks. @velkymx more information would be needed or this ticket is no longer relevant. 

![screen shot 2019-02-19 at 11 02 48 am](https://user-images.githubusercontent.com/29641725/53040352-0d728c80-3436-11e9-947c-3efe898070e3.png)
![screen shot 2019-02-19 at 11 03 46 am](https://user-images.githubusercontent.com/29641725/53040353-0d728c80-3436-11e9-9fd7-0eacf72e44d2.png)
